os-autoinst-obs-auto-submit

```diff
@@ -99,9 +99,17 @@ generate_dep_changelog() {
 osc="${osc:-"$prefix osc"}"
 
 TMPDIR=${TMPDIR:-$(mktemp -d)}
+failed_packages=
 (cd "$TMPDIR"
 for package in os-autoinst openQA os-autoinst-distri-opensuse-deps; do
+    echo "Considering package $package"
     $osc service wait "$src_project"/$package
+    failed_results=$($osc results --repo openSUSE_Factory --failed "$src_project" "$package")
```

Comment:
I would start with something simpler to abort the complete run if *any* package within that project fails.

Reply:
I think this way is more consistent with the skipping logic which is already implemented. And in most cases it also make sense to still update openQA even if os-autoinst can not be built (and vice versa).

Reply:
Yes, most cases. But I would go with "worst case" as design guideline